lib/lookup.json

```diff
@@ -504,5 +504,9 @@
   "@nearform/bubbleprof": {
     "maintainers": ["mcollina", "BridgeAR"],
     "prefix": "v"
+  },
+  "got": {
+    "maintainers": ["sindresorhus"],
+    "skip": ["<8", "11"]
```

Comment:
Actually looking at https://github.com/sindresorhus/got/releases does this need
```
    "prefix": "v"
```
?

Reply:
It would be ignored because this module is published from the git repository and in that case we use the commit sha: https://github.com/nodejs/citgm/blob/1d881a76978a550657e58ee772101205fcc42a11/lib/lookup.js#L10-L29

Reply:
It does no harm, so I added it anyway